source/appModules/winword.py

```diff
@@ -54,7 +54,9 @@ class WinwordWordDocument(WordDocument):
 	def _get_description(self) -> str:
 		try:
 			curView = self.WinwordWindowObject.view.Type
-			return f"{ViewType(curView).displayString} {super()._get_description()}"
+			if isinstance(super()._get_description(), str):
+				return f"{ViewType(curView).displayString} {super()._get_description()}".strip()
+			return curView.displayString
```

Comment:
I think an else would make this more clear.

```suggestion
			else:
				return curView.displayString
```

Reply:
Thanks for your review, Luke. I think that else is redundant and I prefer not to add it after a return. This is one of the first things that I learned thanks to an add-on reviewer that fixed it in an add-on created by me, as logic simplification. I think we can leave this like that.